source/_layouts/home.html

```diff
@@ -75,7 +75,7 @@ <h2 class="font-extrabold text-[22px] md:text-[44px] text-black leading-6 md:lea
             <br /> Explore a new universe of possibilities with Home Assistant. Extend and empower your home automations with your imagination.
           </p>
           <div class="mt-11">
-            <button type="button" class="inline-flex leading-[20px] font-bold items-center px-[26px] bg-white  py-[21px] border-2 border-grey-secondary text-[16px] md:text-[24px] rounded-[10px] text-black"> See what you get more <svg class="ml-[25px]" width="20" height="15" viewBox="0 0 20 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <button type="button" class="inline-flex leading-[20px] font-bold items-center px-[26px] bg-white  py-[21px] border-2 border-grey-secondary text-[16px] md:text-[24px] rounded-[10px] text-black" onclick="window.location.href='/integrate';"> See what you get more <svg class="ml-[25px]" width="20" height="15" viewBox="0 0 20 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should that be done with javascript? That feels incorrect?

Reply:
Agreed.
```suggestion
            <a href="/integrate"><button type="button" class="inline-flex leading-[20px] font-bold items-center px-[26px] bg-white  py-[21px] border-2 border-grey-secondary text-[16px] md:text-[24px] rounded-[10px] text-black"> See what you get more <svg class="ml-[25px]" width="20" height="15" viewBox="0 0 20 15" fill="none" xmlns="http://www.w3.org/2000/svg"></a>
```

Reply:
hmm.. changing it to an anchor tag seems to have messed with the styles for it